build/osx/wxiphone.xcodeproj/xcshareddata/xcschemes/static.xcscheme

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "0940"
+   LastUpgradeVersion = "0900"
```

Comment:
Is this intentional? Important? @csomor It it ok to merge this?

Reply:
I just ran the python script. Maybe different Xcode version? Mine is 13.2.

Reply:
It also changed file order and many identifiers in the pbxproj files. I don't know if this is expected or if it is possible to only add/remove the changed scintilla files.